src/v/cluster/controller.cc

```diff
@@ -646,6 +646,7 @@ ss::future<> controller::start(
             std::ref(_config_frontend),
             std::ref(_feature_table),
             std::ref(_roles),
+            std::addressof(_plugin_table),
```

Comment:
i didn't even know one could overload `operator&` i wonder where that is used in practice